app/controllers/items/detail.js

```diff
@@ -434,8 +434,15 @@ export default GoodcityController.extend(
         this.set("selectedDescriptionLanguage", langauge);
       },
 
-      async updateAttribute(name, value, cb = _.noop) {
+      /**
+       * @param {String} name | Name of the field to update
+       * @param {any} value | Value of the field
+       * @param {boolean} isRequired | false by default
```

Comment:
FYI: JSDoc supports [optional params and default args](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values) ;)

The brackets `[]` indicate that it's optional and the `=false` indicates the default value

```javascript
       * @param {boolean} [isRequired=false]
```